src/components/List/ListItem/ListItem.jsx

```diff
@@ -1,25 +1,45 @@
 import React from 'react';
+import { DragSource } from 'react-dnd';
 import classnames from 'classnames';
-import { ChevronUp16, ChevronDown16 } from '@carbon/icons-react';
+import { Draggable16, ChevronUp16, ChevronDown16 } from '@carbon/icons-react';
 import PropTypes from 'prop-types';
 import isEmpty from 'lodash/isEmpty';
 
+import { EditingStyle, DropLocation } from '../../../utils/DragAndDropUtils';
 import { settings } from '../../../constants/Settings';
 
+import ListTarget, { TargetSize } from './ListTarget';
+
 const { iotPrefix } = settings;
 
-// internal component
-const ListItemWrapper = ({ id, isSelectable, onSelect, selected, isLargeRow, children }) =>
-  isSelectable ? (
+export const ItemType = 'ListItem';
+
+const ListItemWrapper = ({
```

Comment:
With this added code, I think its time to make a separate component file for `ListItemWrapper`